doc/lsst.afw.cameraGeom/cameraGeom.rst

```diff
@@ -0,0 +1,147 @@
+
+.. _section_Camera_Geometry_Utilities:
+
+Camera Geometry Utilities
+=========================
+
+There are a few utilities available for visualizing and debugging Camera objects.
+Examples of available utility methods are: display a particular amp, display an assembled sensor, display a the full camera modaic, plot the sensor boundaries with a grid of test points in :ref:`FOCAL_PLANE <CameraGeom_FOCAL_PLANE>` coordinates.
+An example of how to use the utilities to visualize a camera is available in the obs_lsstSim package as ``$OBS_LSSTSIM_DIR/bin/displayCamera.py``.
+
+.. _section_Camera_Coordinate_Systems:
+
+Camera Coordinate Systems
+=========================
+
+The cameraGeom package supports the following camera-based 2-dimensional coordinate systems, and it is possible to add others:
+
+.. _CameraGeom_FOCAL_PLANE:
+
+``FOCAL_PLANE``
+  Position on a 2-d planar approximation to the focal plane (x,y mm).
+  The origin and orientation may be defined by the camera team, but we strongly recommend that the origin be on the optical axis and (if using CCD detectors) that the X axis be aligned along CCD rows.
+  Note: location and orientation of detectors are defined in a 3-d version of ``FOCAL_PLANE`` coordinates
+  (the z axis is also relevant).
+
+.. _CameraGeom_FIELD_ANGLE:
+
+``FIELD_ANGLE``
+  Angle of a principal ray relative to the optical axis (x,y radians).
+  The orientation of the x,y axes is the same as ``FOCAL_PLANE``.
+
+.. _CameraGeom_PIXELS:
+
+``PIXELS``
+  Nominal position on the entry surface of a given detector (x, y unbinned pixels).
+  For CCD detectors the x axis *must* be along rows (the direction of the serial register).
+  This is required for our interpolation algorithm to interpolate across bad columns.
+
+.. _CameraGeom_ACTUAL_PIXELS:
+
+``ACTUAL_PIXELS``
+  Like ``PIXELS``, but takes into account pixel-level distortions (deviations from the nominal model of uniformly spaced rectangular pixels).
+
+.. _CameraGeom_TAN_PIXELS:
+
+``TAN_PIXELS``
+  Is a variant of ``PIXELS`` with estimated optical distortion removed.
+  ``TAN_PIXELS`` is an affine transformation from ``FIELD_ANGLE`` coordinates, where ``PIXELS`` and ``TAN_PIXELS`` match at the center of the pupil frame.
+
+.. _section_CameraGeom_Basic_Usage:
+
+Basic Usage
+===========
+
+The file `examples/cameraGeomExample.py <https://github.com/lsst/afw/blob/master/examples/cameraGeomExample.py>`_ shows some basic usage of the cameraGeom package.
+
+.. _section_CameraGeom_Objects:
+
+Objects
+=======
+
+The cameraGeom package contains the following important objects; unless otherwise noted, all are available in both C++ and Python:
+
+.. _subsection_CameraGeom_Camera:
+
+Camera
+------
+
+Camera is a collection of :ref:`Detectors <subsection_CameraGeom_Detector>`.
```

Comment:
Please make Camera a link to the class documentation. I think it would suffice to put the following at the top of this file:
```
.. py:currentmodule:: lsst.afw.cameraGeom
```
and then use ``` `Camera` ``` to get a link (similarly for the other cameraGeom objects mentioned her).

Reply:
Thanks for that: I wasn't aware of `py:currentmodule`. That cleaned up this file quite a bit.